### ISSUE TITLE
[plugin-svelte] Can not find App.css.js

### ISSUE THREAD

**Issue by rof20004** [pikapkg/create-snowpack-app#156]
I created a project with this command:

```
npx create-snowpack-app new-dir --template @snowpack/app-template-svelte --use-yarn
```

When I use: `yarn build` it generates build folder.

I execute this inside build folder: `http-server -p 3000`

When I load the page I got this error and not show anything: 

`GET http://localhost:3000/_dist_/App.css.js net::ERR_ABORTED 404 (Not Found)`

What I am doing wrong?

**Comment by drwpow** [comment]
Hey @rof20004, we’re actually working on this problem right now. This is definitely a bug; you’re not doing anything wrong. We plan on having a fix for this soon!

**Comment by drwpow** [comment]
For now, it seems `snowpack dev` is working fine, so I’d recommend using that until we get the `.svelte` build issue fixed in a release.

**Comment by FredKSchott** [comment]
Fixed in #557 and released in v2.6.2.
